project.clj

```diff
@@ -53,31 +53,76 @@
   ;                                 :password
   ;                                 [:env/datomic_password]}}
   :source-paths ["src"]
-  :resource-paths ["resources" "resources-index/prod"]
+  :resource-paths ["resources"]
   :target-path "target/%s"
+
+  ;; Should be doing https://github.com/cemerick/clojurescript.test/issues/97 instead, but I can't figure it out so we turn off clean protection instead
+  :clean-targets ^{:protect false} [:target-path :compile-path "resources/public/js/compiled"]
+
   :main ^:skip-aot dat.sys.run
-  :repl-options {:init-ns user}
-  :cljsbuild {:builds {:client {:source-paths ["src/dat/sys/client"
-                                               "checkouts/datview/src"
-                                               "checkouts/datsync/src"
-                                               "checkouts/datreactor/src"
-                                               "checkouts/datspec/src"]
-                                :compiler {:output-to "resources/public/js/app.js"
-                                           :output-dir "dev-resources/public/js/out"}}}}
+  :cljsbuild {:builds
+              {:client
+               {:source-paths ["src/dat/sys/client"
+                               "dev"
+                               ;;"checkouts/datview/src"
```

Comment:
Did these give you problems? If so perhaps add a comment line suggesting users can uncomment if they want to hack on one of the other Dat\* libs.